openstack/designate/templates/seeds.yaml

```diff
@@ -29,6 +29,7 @@ spec:
   - {{ .Release.Namespace }}/domain-neo-seed
   - {{ .Release.Namespace }}/domain-s4-seed
   - {{ .Release.Namespace }}/domain-wbs-seed
+  - {{ .Release.Namespace }}/monsoon3/domain-cis-seed
```

Comment:
`{{ .Release.Namespace }}/domain-cis-seed`

"monsoon3" is the Namespace, would be duplicated